src/backend/users/views.py

```diff
@@ -105,4 +109,76 @@ def get(self, request, **kwargs):
             "useCommuterRail": user_profile["travel_mode"] == "BTE",
             "voucherNumber": user_profile["voucher_number"],
         }
+        return content
+
+    def get(self, request, **kwargs):
+        user = User.objects.get(username=request.user)
+        serializer = UserSerializer(user)
+
+        content = self.repackage_for_frontend(serializer.data)
+
         return Response(content)
+
+    def put(self, request, *args, **kwargs):
+        data = request.data
+        updated_profile = UserProfile.objects.get(user=User.objects.get(username=request.user))
+
+        # save new/changed destinations
+        for destination in data["destinations"]:
```

Comment:
Thanks for reviewing! I'm inclined towards the first option, especially because we're not expecting a lot of activity. It just feels cleaner and easy to read.